source/components/molecules/FilePicker/imageUpload.ts

```diff
@@ -1,15 +1,64 @@
-import { Alert } from "react-native";
-import type { ImageOrVideo } from "react-native-image-crop-picker";
+import { Alert, Linking, Platform } from "react-native";
 import ImagePicker from "react-native-image-crop-picker";
 import uuid from "react-native-uuid";
 
-import type { Image } from "../ImageDisplay/ImageDisplay";
+import type {
+  ImageOrVideo,
+  PickerErrorCode,
+} from "react-native-image-crop-picker";
+import { splitFilePath } from "../../../helpers/FileUpload";
 
 import type { AllowedFileTypes } from "../../../helpers/FileUpload";
-import { splitFilePath } from "../../../helpers/FileUpload";
+import type { Image } from "../ImageDisplay/ImageDisplay";
+
+interface ImageUploadError {
+  code: PickerErrorCode;
+}
+
+const NO_IMAGE_UPLOAD_PERMISSION_CODES = [
```

Comment:
We can type this as `PickerErrorCode[]`.

Reply:
Done